content/en/platform/corda/4.11/enterprise/node/collaborative-recovery/ledger-recovery/overview.md

```diff
@@ -0,0 +1,75 @@
+in subsequent transactions with other nodes in the network, yet.
+{{< /note >}}
+
+## Transaction distribution records
+
+Ledger Recovery builds on the foundations established in [Two Phase Finality]({{< relref "../../../two-phase-finality.md" >}}),
+where recovery metadata is stored for transactions at both the sender's and receiver's side of a transaction flow.
+
+For any given transaction, the sender's side stores one or more `SenderDistribution` records in its local
+`sender_distribution_records` database table. There is one `SenderDistribution` record for each receiver peer of a transaction.
+Receiver peers include any participants and/or observers to the transaction.
+
+A `SenderDistribution` record contains the following transaction metadata:
+* Transaction ID
+* Receiver peer ID (the secure hashed value of `CordaX500Name` using SHA256)
+* Receiver `StatesToRecord` value
+
+{{< note >}}
+The `SendTransactionFlow` has been enhanced to infer the value for receiver `StatesToRecord` based on the type of sessions passed into its constructor:
+* `val participantSessions: Set<FlowSession>` defaults to `ONLY_RELEVANT`
+* `val observerSessions: Set<FlowSession>` defaults to `ALL_VISIBLE`
+{{< /note >}}
+
+Upon storing the `SenderDistribution` records for a transaction, the sender node also generates a single `ReceiverDistribution` record.
+
+A `ReceiverDistribution` record contains the following transaction metadata:
+* Transaction ID
+* Sender peer ID (the hashed value of `CordaX500Name`)
+* A distribution list comprising two sections:
+  * An AES-encrypted map of the peer IDs of all receiving peers for the transaction and their associated `StatesToRecord` value.
+  * A tamper-proof cleartext `senderRecordedTimestamp`, indicating when the sender records were generated. The same
+    timestamp will be used upon storing the `ReceiverDistribution` record at each of the receiving node peers. This is
+    required to enable synchronized record comparisons across peers when performing transaction recovery.
+
+The generated `ReceiverDistribution` record is shared with all receiving peer sessions (both participants and observers)
+to the transaction, which stores it in their local `receiver_distribution_records` database table.
+
+{{< note >}}
+A receiver cannot decrypt the actual contents of the distribution list within that record but can share it back to the sending node upon recovery.
+The Corda X.500 name of all peers stored in a Ledger Recovery record use an undecipherable collision-free representation.
+{{< /note >}}
+
+Both sender's and receiver's distribution records use the same composite key type for uniquely storing records. The `PersistentKey` contains the following fields:
+* The transaction ID
+* PartyId of flow peer (a secure hash stored as a String)
+* Timestamp (Instant)
+* Timestamp discriminator (Int)
+
+The timestamp discriminator allows for storing of records generated at the same time even for the same transaction.
```

Comment:
I'd move this up to be a description as part of the last bullet.

Reply:
Moved